index.html

```diff
@@ -0,0 +1,189 @@
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </div>
+      
+  
+      <div class="main-section">
+        <div class="heading">
+          <p class="header-paragraph">"Computer Workshop"</p>
+          <h1>COMPUTER MAINTENANCE AND SOFTWARE INSTALLATION</h1>
+        </div>
+    
+        <div class="header-text">
+          <p class="pad-class">We provide basic computer maintenance services and we also install softwares of all kinds
+            both system and application softwares
+          </p>
+        </div>
+    
+        <div class="timing">
+          <h3>2023.01.02(MON) ~ 15(TUES)</h3>
+          <p>Curated by Higher Grounds Technological Center</p>
+        </div>
+  
+      </div>  
+    </div>
+
+    <div class="enclose"></div>
+
+    <div class="program">
+      <div class="section-text">
+        <h4>What we do</h4>
+        <div class="program-line"></div>
+      </div>
+
+      <div class="card-container">
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            computer
+          </span>
+            <p class="card-margin technology">Repairs</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            tools_installation_kit
+          </span>
+            <p class="card-margin technology">Install</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            quiz
+            </span>
+            <p class="card-margin technology">Consult</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            point_of_sale
+            </span>
+            <p class="card-margin technology">Sales</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            cast_for_education
+            </span>
+            <p class="card-margin technology">Training</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+      </div>
+      <a href="" class="more-link">See The Whole Services We Offer</a>
+
+
+
+      <button class="btn">Sign up for business</button>
+
+    </div>
+
+    <div class="client-section">
+      <h4>What our clients say...</h4>
+      <div class="program-line"></div>
+
```

Comment:
I noticed your second section of your home page doesnt have a background image. It would be nice if you adde one ill attach the image to use for section. 

Please kindly use this image as your back ground:

![repeat-bg](https://user-images.githubusercontent.com/79506105/210611497-1a0b09dc-4186-46a8-967a-19320f702ce3.png)

This is your design: 

![home2](https://user-images.githubusercontent.com/79506105/210611585-f3634f82-ddeb-4f54-b251-7e14671db45b.png)

Please kindly follow this design:

![Behance 2](https://user-images.githubusercontent.com/79506105/210611735-e189a050-d098-4b87-9812-52cceae3e4ad.png)